appengine/standard/endpoints/backend/app.yaml

```diff
@@ -3,14 +3,25 @@ threadsafe: true
 api_version: 1
 
 handlers:
-# The endpoints handler must be mapped to /_ah/spi.
-# Apps send requests to /_ah/api, but the endpoints service handles mapping
-# those requests to /_ah/spi.
-- url: /_ah/spi/.*
+# The endpoints handler must be mapped to /_ah/api.
+- url: /_ah/api/.*
   script: main.api
 
 libraries:
 - name: pycrypto
   version: 2.6
-- name: endpoints
-  version: 1.0
+
+# Endpoints 1.1 uses background threads for caching and reporting to service
+# management and service control.
+manual_scaling:
+  instances: 1
+
+beta_settings:
+  use_endpoints_api_management: true
+  endpoints_swagger_spec_file: echo-v1_swagger.json
+
+env_variables:
+  #TODO: The Endpoints service name.
```

Comment:
Don't use #TODO, just say "Replace with your endpoints service name, for example your-service.appspot.com"